expressEngine/logger/location.js

```diff
@@ -0,0 +1,18 @@
+const getLocation = (stepInStack = 1) => {
+  try {
+    throw new Error('Log stack');
+  } catch (e) {
+    try {
```

Comment:
Can we use try block inside catch block?

Reply:
yes we can use, reference- https://social.msdn.microsoft.com/Forums/vstudio/en-US/9981167d-5c82-4d7f-8c83-5348976d940a/try-block-inside-catch-block?forum=clr

Reason why I am using-
whenever stack is full it should throw error.